src/model/saveScore.js

```diff
@@ -0,0 +1,32 @@
+const saveScore = (gameType, userName, userScore) =>{
+
+    const scoreTable = [];
+
+    if(getScoresForGameType(gameType) !== null){
+        scoreTable.push(JSON.parse(localStorage.getItem(gameType)));
+    }
+
+    const scoreObject = {
+        'name': userName, 
+        'score': userScore
+    };
+
+    scoreTable.push(scoreObject);
+
+    let userScore_serialized = JSON.stringify(scoreTable);
+
+    localStorage.setItem(gameType, userScore_serialized);
+
+}
+
+export default saveScore;
+
+
+export const getScoresForGameType = (gameType)  => {
+
+    const table = [];
+    table.push(JSON.parse(localStorage.getItem(gameType)));
+
+    return table;
+
+}
```

Comment:
Widzę, że localStorage chyba nie musisz mockować na potrzeby tego projektu.
Spróbuj bez mockowania normalnie przetestować funkcje.

Żeby wyczyścić local storag w pliku z testem to możesz użyć 
![image](https://user-images.githubusercontent.com/41823689/144721075-e38cd4c7-3a1a-4c81-a35c-3dcc72d6b8dd.png)